File: examples/trending-news-statistics.php
<?php

/*

	File: trending-news-statistics.php
	Endpoint: public/trending-news-statistics
	Repo: https://github.com/executium/trending-historical-cryptocurrency-news
	License: https://github.com/executium/trending-historical-cryptocurrency-news/LICENSE

*/

# Include the executium class
include '../src/Executium.trendingnews.Class.php';

# Declare your key and secret
$obj = new ExecutiumTrendingNews();

# Parameters required
$payload=array();

# Send Request
$result = $obj->trending_news_statistics($payload);

# Result
echo '<pre>';
	echo "\n<strong>Returned</strong>\n\n";
	print_r($result);
echo '</pre>';

?>

File: python-sdk/examples/trending-news-statistics.py
'''

	Endpoint: public/trending-news-statistics
	Name: Trending News Statistics

'''
import sys
import json

sys.path.insert(1, '../src')

# Include config and class
from config import key, secret
from Class import ApiAccess

# Declare your key and secret
obj = ApiAccess(key, secret)

# Declare endpoint
endpoint = 'public/trending-news-statistics'

# No parameters required
payload = {}

result = obj.send(endpoint, payload, dict())

print(json.dumps(result['returned'], indent=4))